src/vs-bicep/Bicep.VSLanguageServerClient.IntegrationTests/Utilities/HoverUtility.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Test.Apex.Editor;
+using Microsoft.Test.Apex.VisualStudio.Editor;
+using Microsoft.VisualStudio;
+
+namespace Bicep.VSLanguageServerClient.IntegrationTests.Utilities
+{
+    public class HoverUtility
+    {
+        public static void WaitForQuickInfo(Func<string, bool> funcIsCorrectQuickInfo, IVisualStudioTextEditorTestExtension editor)
+        {
+            string lastQuickInfoText = string.Empty;
+
+            WaitForExtensions.IsTrue(() =>
+            {
+                InvokeQuickInfo();
+
+                try
+                {
+                    IQuickInfoTestExtension? quickInfo = WaitForExtensions.IsNotNull(() => editor.Intellisense.GetActiveQuickInfo(), 1000);
+
+                    if (quickInfo is not null)
+                    {
+                        // The Text property isn't available immediately.
+                        WaitForExtensions.IsTrue(() =>
+                        {
+                            lastQuickInfoText = quickInfo.Text;
+
+                            return !string.IsNullOrEmpty(lastQuickInfoText);
+                        }, TimeSpan.FromMilliseconds(1000));
+
+                    return funcIsCorrectQuickInfo(quickInfo.Text);
+                   }
+                }
+                catch
+                {
+                }
+
+                return false;
+            }, TimeSpan.FromMilliseconds(5000), () => "last QuickInfo text: " + lastQuickInfoText);
+        }
+
+        public static void InvokeQuickInfo()
+        {
+            VsHostUtility.VsHost!.ObjectModel.Commanding.ExecuteCommand(VSConstants.VSStd2KCmdID.QUICKINFO, null);
```

Comment:
nit: extra line

Reply:
Hmmm I don't see an extra line at this location or anywhere in this file. Let me know if I am missing something.